torch/fx/experimental/fx_acc/acc_normalizer.py

```diff
@@ -60,6 +60,9 @@ class NormalizationInfo(NamedTuple):
     custom_mapping_fn: Optional[Callable]
     kwargs_to_move_to_acc_out_ty: Optional[Optional[List[Tuple[str, str]]]]
     needs_shapes_for_normalization: bool
+    # qscheme of Tensor, used to get different fields for quantization
+    # parameters
+    qscheme: Optional[torch.qscheme]
```

Comment:
`qscheme` doesn't feel like a good fit in `NormalizationInfo`. Why do we need this? I saw it mostly just hard coded for some ops.